types/formidable/index.d.ts

```diff
@@ -1,56 +1,447 @@
-// Type definitions for Formidable 1.0.16
-// Project: https://github.com/felixge/node-formidable/
+// Type definitions for formidable 1.1
+// Project: https://github.com/node-formidable/formidable
 // Definitions by: Wim Looman <https://github.com/Nemo157>
+//                 Martin Badin <https://github.com/martin-badin>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
 
+import { IncomingMessage } from 'http';
+import { Stream, Transform, PassThrough } from 'stream';
+import { EventEmitter } from 'events';
+import { type } from 'os';
```

Comment:
IMO, unused (and kind of risky wihtout alias in TS):
```suggestion
```
is that one used somewhere?

Reply:
sorry, no. I will remove that